streamparse/cli/submit.py

```diff
@@ -205,6 +205,17 @@ def submit_topology(name=None, env_name="prod", workers=2, ackers=2,
             if isinstance(inner_shell, ShellComponent):
                 if 'streamparse_run' in inner_shell.execution_command:
                     inner_shell.execution_command = streamparse_run_path
+                    
+    # Additional options
+    additional_options = env_config.get('options', None)
+    if additional_options and isinstance(additional_options, dict):
+        if isinstance(options, dict):
+            additional_options.update(options)
+        options = additional_options
+    if not workers:
+        workers = env_config.get('worker_count', 2)
```

Comment:
Ignore me, I misunderstood what you were doing with `additional_options`.  You still want `env_config` here, since the number of workers and ackers will be environment-specific and not part of the `options` thing dict in the JSON.